doc/userguide/upgrade.rst

```diff
@@ -36,6 +36,10 @@ Upgrading 6.0 to 7.0
 Major changes
 ~~~~~~~~~~~~~
 - Upgrade of PCRE1 to PCRE2. See :ref:`pcre-update-v1-to-v2` for more details.
+- Spaces are accepted in HTTP1 URIs instead of in the protocol version
+
+That is `GET /a b HTTP/1.1` gets now URI as `a b` and protocol as `HTTP/1.1` when
+it used to be URI as `a` and protocol as `b HTTP/1.1`
```

Comment:
This part is a bit confusing... I think that two things could be done.

i) bring the next sentence together to the item list that starts this topic.
ii) adjust the text a bit.

Resulting in:
```
- Spaces are accepted in HTTP1 URIs instead of in the protocol version. That is: 
  `GET /a b HTTP/1.1` gets now URI as `a b` and protocol as `HTTP/1.1`. Before,
  this would result in URI as `a` and protocol as `b HTTP/1.1`.
```

Reply:
Thanks